src/AppBundle/Controller/Report/ReportController.php

```diff
@@ -38,6 +38,7 @@ class ReportController extends AbstractController
         'report-prof-deputy-costs-scco',
         'report-prof-deputy-fixed-cost',
         'prof-deputy-costs-estimate-how-charged',
+        'prof-deputy-management-costs',
```

Comment:
dont need this now, youve already configured it to use prof-deputy-estimate-costs, which is already in the list

Reply:
you could keep it, but youd have to change your api entity to reflect the same group

Reply:
Ah yeah missed this - its dropped now 👍